docs/guide/url_management.md

```diff
@@ -138,6 +138,12 @@ The `scheme` attribute is mandatory and has to correspond to the name of the pro
 You can define URL aliases for individual Content items, for example, when you reorganize the content, and want to provide users with continuity.
 For each URL alias definition the history of changes is preserved, so that users who have bookmarked the URL addresses of content items can still find he information they desire.
 
+!!! note
+
+    Make sure to correctly define languages used by site in configuration (`ezplatform.system.*.languages`). Otherwise redirections for renamed Content
```

Comment:
```suggestion
    Make sure that you correctly define languages used by the site in the configuration 
    (under the `ezplatform.system.*.languages` key). 
    Otherwise, redirections for the renamed Content with translations in multiple
```